mlir/Makefile

```diff
@@ -69,7 +70,8 @@ llvm:
 		-DLLVM_ENABLE_LLD=$(ENABLE_LLD) \
 		-DLLVM_ENABLE_ZLIB=$(ENABLE_ZLIB) \
 		-DLLVM_ENABLE_ZSTD=$(ENABLE_ZSTD) \
-		-DCMAKE_CXX_VISIBILITY_PRESET=$(SYMBOL_VISIBILITY)
+		-DCMAKE_CXX_VISIBILITY_PRESET=$(SYMBOL_VISIBILITY) \
+		$(CMAKE_ARGS)
```

Comment:
Are you able to re-build LLVM/MLIR and MHLO deps from source after adding these flags? :thinking: I'm getting some weird CMake errors with `find_package` LLVM/MLIR and MHLO cmake configs as the result of these changes.